data/helpers.d/mysql

```diff
@@ -90,3 +90,47 @@ ynh_mysql_create_user() {
 ynh_mysql_drop_user() {
     ynh_mysql_execute_as_root "DROP USER '${1}'@'localhost';"
 }
+
+# Create a database, an user and its password. Then store the password in the app's config
+#
+# User of database will be stored in db_user's variable.
+# Name of database will be stored in db_name's variable.
+# And password in db_pwd's variable.
+#
+# usage: ynh_mysql_generate_db user name
+# | arg: user - Owner of the database
+# | arg: name - Name of the database
+ynh_mysql_generate_db () {
```

Comment:
Why not, maybe it's better, it's more distant to ynh_mysql_create_db than the current name.